src/popper/runner_host.py

```diff
@@ -374,6 +374,9 @@ def _create_container(self, cid, step):
             return None
 
         container = container.rsplit()
+        if len(container) < 1:
```

Comment:
should this be `len(container) == 0`? Doing `container[-1]` with a 1-size list works fine